bloxid/v0.go

```diff
@@ -43,17 +43,20 @@ var (
 
 // NewV0 parse a string into a typed guid, return an error
 // if the string fails validation.
-func NewV0(bloxid string) (*V0, error) {
+func NewV0(bloxid, salt string) (*V0, error) {
```

Comment:
the salt should not be an arg to this func, users will need to supply it via WithSalt(salt string)